### PR TITLE
add create new cards

### PR DESCRIPTION
**Pull request by lenzloesche**

### CODE REVIEW

**Review by nikokreissl — APPROVED** [review]
Looking good! 😊
Possibly there are some options to save some lines of code but great for now!

**Comment by lenzloesche** [comment]
> Looking good! 😊 Possibly there are some options to save some lines of code but great for now!

thanks for reading the code.
yes maybe i should add a function that creates an element. i dont know.

File: form.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+const form = document.querySelector('[data-js="form"]');
+const main = document.querySelector('[data-js="main"]');
```
Comment (mpagels) [inline form.js:2]: Do you need this query? I didn't see that you're using it anywhere.

File: form.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+const inputTag1 = document.querySelector('[data-js="tag1"]');
+const inputTag2 = document.querySelector('[data-js="tag2"]');
+const inputTag3 = document.querySelector('[data-js="tag3"]');
+
+form.addEventListener("submit", (event) => {
+  event.preventDefault();
+
+  const formData = new FormData(event.target);
+  const dataObject = Object.fromEntries(formData);
+  console.log(dataObject);
```
Comment (mpagels) [inline form.js:14]: A tiny little `console.log` is here. ;-P 
Highly recommend removing it in production code.

File: form.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    tag2.textContent = "#" + inputTag2.value;
+    tagList.append(tag2);
+  }
+
+  if (inputTag3.value) {
+    const tag3 = document.createElement("li");
+    tag3.classList.add("card__tag", "shadow");
+    tag3.textContent = "#" + inputTag3.value;
+    tagList.append(tag3);
+  }
```
Comment (mpagels) [inline form.js:67]: I would recommend creating the `ul` element after checking if a `tag` was provided. If not, you have an empty `ul` element on your page, without a `Li` element. ;-)

File: form.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    tag2.classList.add("card__tag", "shadow");
+    tag2.textContent = "#" + inputTag2.value;
+    tagList.append(tag2);
+  }
+
+  if (inputTag3.value) {
+    const tag3 = document.createElement("li");
+    tag3.classList.add("card__tag", "shadow");
+    tag3.textContent = "#" + inputTag3.value;
+    tagList.append(tag3);
```
Comment (mpagels) [inline form.js:66]: This is repetitive code. You could get rid of a lot of redunaned code, if you use a function, that would do all the work for you.

See my example:

```js
function addTagToTagList(inputTag) {
  if (inputTag.value) {
    const tag = document.createElement("li");
    tag.classList.add("card__tag", "shadow");
    tag.textContent = "#" + inputTag.value;
    tagList.append(tag);
  }
}
addTagToTagList(inputTag1)
addTagToTagList(inputTag2)
addTagToTagList(inputTag3)
```

**Review by mpagels — COMMENTED** [review]
Good work here. Found some small improvements. Pls have a look. Keep up the good work!

**Pull request merged**